### ISSUE TITLE
Inquiry on Performance

### ISSUE THREAD

**Issue by HonghuiBao2000** [anananan116/MBGen#10]
Dear author,

Thank you for providing the source code and data — it’s a very insightful piece of work!

While trying to reproduce the results, I downloaded the Retail dataset from the link you provided and ran the code directly with almost no changes to the configuration.

```bash
python run.py --config=./config/SID/retail/main.yaml
````

After training with one GPU for 23 epochs, I obtained the results.

> Target Evaluation Metrics:
Test Loss(target): 2.5150638202826183
Recall@5(target): 0.0625
Recall@10(target): 0.08489583333333334
NDCG@5(target): 0.04451165784399923
NDCG@10(target): 0.05178511026179444

However, the performance I got seems to differ significantly from the results reported in Table 3, which left me a bit confused — did I miss anything important during this process?

Apologies if this is a naive question. I really appreciate any help you can provide!!!

**Comment by anananan116** [comment]
Hi, thank you for your interest in this project. To help to better understand the issue, could you please provide more details?

When you say "almost no changes to the configuration" can you indicate what you changed if you changed anything? Could you also check the training loss reported since the test loss seems surprisingly high.
